### PR TITLE
Remove azure-blobstorage-artifactProvider common package

### PR DESCRIPTION
**Pull request by damccorm**
**Description**: Move azure-blobstorage-artifactProvider into the Jenkins task which is the only consumer (outside of the JavaToolInstaller which had a dead reference to it in a test)

**Documentation changes required:** N

**Added unit tests:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/JenkinsDownloadArtifactsV1/AzureStorageArtifacts/blobservice.ts [hunk 1]
```diff
@@ -16,27 +13,6 @@ export class BlobService {
         this._host = host;
     }
 
-    public async uploadBlobs(source: string, container: string, prefixFolderPath?: string, itemPattern?: string): Promise<string[]> {
```
Comment (stephenmichaelf) [inline Tasks/JenkinsDownloadArtifactsV1/AzureStorageArtifacts/blobservice.ts:19]: We moved the file but don't need this? Is it dead code?

**Comment by stephenmichaelf** [comment]
Missing bump of `task.json` and `task.loc.json`.

File: Tasks/JenkinsDownloadArtifactsV1/AzureStorageArtifacts/blobservice.ts (hunk 1 above)
Comment (damccorm) [reply Tasks/JenkinsDownloadArtifactsV1/AzureStorageArtifacts/blobservice.ts:19]: Yeah. There was actually a fair amount of dead code here, I didn't move several files

**Comment by bishal-pdMSFT** [comment]
@damccorm this common module is also used in AzureVmssDeployment task and that will break. https://github.com/microsoft/azure-pipelines-tasks/blob/8187e587ee27779376958f761ba9243d058758ac/Tasks/AzureVmssDeploymentV0/make.json#L10

**Comment by damccorm** [comment]
@bishal-pdMSFT that is azure-blobstorage-artifactProvider-v2, not just azure-blobstorage-artifactProvider. They're different packages

**Comment by bishal-pdMSFT** [comment]
Ah yes

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**